apps/site/src/content/blog/astro-theme-provider.mdx

```diff
@@ -0,0 +1,84 @@
+* Add themes to existing projects
+* Configure a theme by passing arguments
+
+Although theme integrations are more user-friendly than project templates, it comes at the cost of being harder to create. They require a lot of boilerplate and a lower level knowledge of Astro and Vite making it inaccessible for a lot of developers.
+
+Astro Theme Provider solves this by making it easier for theme authors to create theme integrations without having to write any integration code. Themes can be authored similar to a normal Astro project and be exported for others to use with a single function call.
+
+```
+package/
+├── public
+├── src/
+│   ├── assets
+│   ├── components
+│   ├── css
+│   ├── layouts
+│   └── pages
+├── index.ts
+└── package.json
+```
+
+```ts
+// /package/index.ts
+import { z } from 'astro/zod'; 
+import defineTheme from 'astro-theme-provider';
+
+export default defineTheme({
+    schema: z.object({
+        title: z.string()
+    })
+})
+```
+
+After creating your theme, it can be used inside an Astro project as an integration:
+
+```ts
+import { defineConfig } from "astro/config";
+import myTheme from "my-theme";
+
+export default defineConfig({
+    integrations: [
+        myTheme({
+            config: {
+                title: "Welcome to my theme!",
+            },
+        }),
+    ],
+});
+```
+
+## Using the right tool for the job
+
+Theme integrations may be easier to use, but there are some cases where using a traditional project template makes more sense. For example, themes that require modifications to the source code, like a SAAS starter, are better suited as project templates. This can also make theme integrations harder to develop. Since users can't edit the source code themselves, authors have to create more verbose pages and components to support customizations from the user.
+
+Ultimately, it is up to the theme author to weigh the advantages and disadvantages of both approaches and choose the one that works best for their project.
+
+## Real world example
+
+The creation of Astro Theme Provider was inspired by Starlight, Astro's official documentation theme. Starlight is a theme integration that allows you to build a full documentation website with a config object and some markdown. Its ease of setup and use has made Starlight one of the most popular themes for Astro. Its success proves that theme integrations are a powerful pattern that have been under utilized in the Astro ecosystem.
+
+## What’s Next
+
+We are looking for help! Astro Theme Provider is in a beta stage, you can create themes with it, but it has not been officially released. The project is being opened up to the community so that others can get involved. If you are looking to help, you can check out the contributing guide. We encourage contributions of any kind! If you have any feedback or suggestions for the project, you can join the Discord channel or use GitHub Issues and Discussions.
```

Comment:
```suggestion
We're excited to share that Astro Theme Provider is in beta as of today. What's next? - We're working on improving the experience and shaping a stable version, but we need your help! We encourage contributions of any kind! You can check out the Project on [GitHub](https://github.com/astrolicious/astro-theme-provider) and join us on [Discord](https://chat.astrolicious.dev)!
```

I think you did a great job, but some sentences feel having the same point. Above my suggestion, but feel free to not use it, but make your own version :)